docs/deep/instanseg.md

```diff
@@ -44,56 +46,57 @@ The InstanSeg user interface
+Select the dropdown box to see the available models. Options with a cloud icon will need to be downloaded, as they are not local to your machine. To do this, select the model and click the download button to fetch them. If you have local models in your directory, you can also select these from the dropdown box. Be sure to select the relevant model for the type of image you are working with (unless you are being experimental!). The model being used here is for brightfield images and is being used on a haematoxylin and DAB stain, but it could be used on images captured using other stains.
 
 :::{note}
 Please note that internet in needed to download the models.
 :::
 
 ### 3. Create or select an annotation
 
-Create an annotation or select a pre-existing annotations/tiles you wish to run the model on. It's recommended that if this is the first time running InstanSeg to keep the annotation smaller to test the processing speed before running it on a larger region. This might take some time, depending on your computers processing speed.
+Create an annotation or select pre-existing annotations/tiles/TMA cores you wish to run the model on. It's recommended to keep the annotation smaller if this is the first time running InstanSeg. This allows you to test the processing speed before running it on a larger region. This might take some time, depending on your computer's processing speed and whether you're using a GPU.
 
 ### 4. Run the model
 
-When you select run, InstanSeg will check for PyTorch. If this is not on your machine it will download it for you (this may well be > 100 MB, so may take a while). Once this is done the model will run and you will see the results in the viewer.
+When you click `Run`, InstanSeg will check for PyTorch. If this is not on your machine it will download it for you (this may well be > 100 MB, so may take a while). Once this is done, the model will run and you will see the results in the viewer.
 
 :::{figure} images/instanseg_running.png
 :class: shadow-image large-image
 
-The running the InstanSeg model
+Running InstanSeg
 :::
 
 ### 5. Viewing Results
 
-The results will be displayed in the viewer. The cells detections can be turned on or off using the show/hide detection objects button in the toolbar. Additionally to help distinguished the cells an overlay can be placed over the cells using the fill/unfill detection objects button and the opacity adjusted using the slider also in the toolbar.
+The results will be displayed in the viewer. The visibility of detections can be turned on or off using the show/hide detection objects button in the toolbar. Additionally, using the fill/unfill detection objects button and the opacity slider in the toolbar can help distinguish the cells.
 
 :::{figure} images/instanseg_bf_results.png
```

Comment:
I think we should stick with one image, not switch between fluorescence and brightfield and back.